README.md

```diff
@@ -43,6 +40,8 @@ Install the [Soil](https://github.com/roots/soil) plugin to enable additional fe
 * Cleaner output of `wp_head` and enqueued assets
 * Root relative URLs
 * Nice search (`/search/query/`)
+* HTML5 Boilerplate's Google CDN jQuery snippet
+* HTML5 Boilerplate's Google Analytics snippet
```

Comment:
I think one of these should link to H5BP since the link to them was removed above.